packages/sdk/odahuflow/sdk/gppi/entrypoint_invoke.py

```diff
@@ -37,6 +36,9 @@
 _logger = logging.getLogger(__name__)
 
 ODAHUFLOW_MODEL_LOCATION_ENV_VAR = 'MODEL_LOCATION'
+MODEL_ENTRYPOINT_ENV = 'MODEL_ENTRYPOINT'
+
+args = None
```

Comment:
Why did you add the global variable?

Reply:
Actually I didn't add it . It already was, but previously args are parsed in import __main__ section of script that is module scope, but now they are parsed in ::main():: function and therefore they become local variable and all other functions that had access to ::args:: – lose it access